explorer/app/components/Project/components/CollaboratingOrganizations/collaboratingOrganizations.tsx

```diff
@@ -7,7 +7,7 @@ import { SectionDetailsEmpty } from "../Section/components/SectionDetailsEmpty/s
 import { Section } from "../Section/section";
 
 // Styles
-import { Sup } from "../../../Citations/Citations.styles";
+import { Sup } from "../Sup/Sup.styles";
```

Comment:
`Citations` unused and subsequently deleted. `Sup` style in use and refactored to `Project` folder.